src/ncp/ncp_spi.cpp

```diff
@@ -270,8 +271,6 @@ ThreadError NcpSpi::PrepareNextSpiSendFrame(void)
     uint16_t frameLength;
     uint16_t readLength;
 
-    VerifyOrExit(!mSending, errorCode = kThreadError_Busy);
```

Comment:
The test for `!mSending` now happens in `PrepareTxFrame()` before calling this function, so it's not really needed here anymore.